modules/core/src/main/scala/org/tesselation/config/Config.scala

```diff
@@ -26,22 +26,23 @@ object Config {
 
   def default[F[_]](environment: AppEnvironment): ConfigValue[F, AppConfig] =
     (
-//      env("CL_DUMMY_SECRET").default("foo").secret,
+      env("CL_EXTERNAL_IP").default("127.0.0.1"),
```

Comment:
Maybe we can use regular config for that? Env variables with `CL_` prefix would just override config properties resulting in the same behavior but in a uniform way.  https://github.com/lightbend/config#optional-system-or-env-variable-overrides

Reply:
We can discuss this and change if needed

Reply:
Ok